source/jormungandr/tests/bragi_autocomplete_tests.py

```diff
@@ -825,11 +980,13 @@ def get_autocomplete(query):
                 assert response_from['poi']['properties']["amenity"] == "bicycle_rental"
                 assert response_from['poi']['properties']["capacity"] == "20"
                 assert response_from['poi']['properties']["ref"] == "12"
-                assert response_from['poi']['address']['id'] == "179077655"
+                assert response_from['poi']['address']['id'] == "addr:8.9809147;42.561667"
```

Comment:
This is not a correct address id: you don't get a prefix from kraken autocomplete
This was also part of the JIRA comment (the minor part of the 3)